staging/src/k8s.io/apiextensions-apiserver/pkg/controller/establish/establishing_controller.go

```diff
@@ -99,6 +99,7 @@ func (ec *EstablishingController) processNextWorkItem() bool {
 
 	err := ec.syncFn(key.(string))
 	if err == nil {
+		ec.queue.Forget(key)
```

Comment:
@yue9944882 thanks for catching this! :+1: 

This lgtm, but I'm wondering if we should also check if `key` is  indeed a string in `key.(string)`. If not, I think this will panic or do something crazy like going into a loop of processing something that's invalid.

Reply:
But again, I don't think key will ever not be string. So I think it should be fine.